tests/integration/test_git_integration.py

```diff
@@ -101,7 +101,7 @@ def test_setup_ssh_for_git_host_create_known_host_and_env_var():
 
         private_key = ssh_dir / "id_rsa"
         assert private_key.exists() is True
-        assert private_key.read_text() == "MY_PRIVATE_KEY"
+        assert private_key.read_text() == "MY_PRIVATE_KEY\n"
```

Comment:
Is it worth adding in a version of this test where,

```python
os.environ[SSH_PRIVATE_KEY_ENV_VAR] = "MY_PRIVATE_KEY\n"
```

Would this be overkill?

Reply:
Testing an individual 'If' statement is overkill. Our other tests will fail if it cannot handle the '/n' at the end as they all read from the file programatically and therefore have the line break at the end.

Reply:
You're right.